minetest/minetest
# Minetest mod path

This patch adds an environment variable "MINETEST_MOD_PATH".  It functions like "MINETEST_SUBGAME_PATH", but for individual mods.  This is necessary for installing minetest mods with Guix (see <https://issues.guix.gnu.org/49828>), otherwise Minetest won't find the mods.

I tested that installing mods from ContentDB with the built-in installer still works, even if "MINETEST_MOD_PATH" is set.

I don't know why GitHub shows 169 commits as I only added one.

## To do

This PR is Ready for Review, except for the "169 commits" issue.

## How to test

Set "MINETEST_MOD_PATH" to "/location/of/some/mod:/location/of/some/other/mod", start Minetest, create a world, select the two mods, start the game, verify the two mods are actually loaded.  Test that mods can still be installed with the built-in installer, and they are installed in ~/.minetest/mods, not  /location/of/some/mod or /location/of/some/other/mod.
